scripts/queries/netobserv_prometheus_queries.yaml

```
@@ -169,3 +169,15 @@
 # total working set memory by all Kafka pods - per pod is not included here as that is collected above
 - query: sum(container_memory_working_set_bytes{namespace="netobserv", container="", pod=~"kafka.*"})
   metricName: workingsetKafkaTotals
+
+###########################
+# Total NetObserv metrics #
+###########################
+
+# Total NetObserv CPU Usage, assuming Loki and Kafka also runs in netobserv NS
+- query: (sum(pod:container_cpu_usage:sum{namespace=~"netobserv|openshift-netobserv-operator"}) or on() vector(0))
```

it doesn't seem to capture netobserv-privileged?
=>
```suggestion
- query: (sum(pod:container_cpu_usage:sum{namespace=~"netobserv.*|openshift-netobserv-operator"}) or on() vector(0))
```

---

thanks for catching this.